common/src/api/external/error.rs

```diff
@@ -144,11 +144,11 @@ pub enum LookupType {
     ByName(String),
     /// a specific id was requested
     ById(Uuid),
-    /// a session token was requested
-    BySessionToken(String),
```

Comment:
This is not used. What's weird is it was introduced in #814, but it is also not used there.

Reply:
Thanks for removing this. I'm not sure how much sense it made in this context to begin with.